lib/types.js

```diff
@@ -61,8 +59,25 @@ TInt.prototype = {
 
 exports.TInt = TInt;
 
-function TTime (value) { this.value = value; }
+function TLong (value) { this.value = value; }
+TLong.prototype = {
+    typetag: 'h',
+    decode: function (data) {
+        if (data.length < 8) {
+            throw new ShortBuffer('int', data, 8);
+        }
 
+        this.value = jspack.Unpack('>i', data.slice(0, 8))[0];
+        return data.slice(8);
+    },
+    encode: function (buf, pos) {
+        return pack('>i', buf, pos, [ this.value ]);
```

Comment:
this will need to be either `l` for signed or `L` for unsigned. This encoding will pack it as an int